python/src/trezorlib/ethereum.py

```
@@ -220,24 +223,38 @@ def sign_tx(
+                response = session.call(messages.EthereumTxAck(data_chunk=chunk))
+        elif isinstance(response, messages.EthereumDefinitionRequest):
+            # We are being asked for a function definition.
+            if definition_provider is not None:
+                ack = definition_provider(response)
+            else:
+                ack = messages.EthereumDefinitionAck(
+                    definitions=None, func_definition=None
+                )
+            response = session.call(ack)
+        else:
+            assert False, "Unknown Ethereum request"
```

Ruff (0.15.10)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)